xbmc/video/VideoDatabase.cpp

```diff
@@ -2432,6 +2503,9 @@ int CVideoDatabase::SetDetailsForEpisode(const std::string& strFilenameAndPath,
     AddActorLinksToItem(idEpisode, MediaTypeEpisode, "director", details.m_director);
     AddActorLinksToItem(idEpisode, MediaTypeEpisode, "writer", details.m_writingCredits);
 
+    // add ratings
+    details.m_iIndexRating = AddRatings(idEpisode, MediaTypeEpisode, details.m_rating, details.m_strDefaultRating);
```

Comment:
We don't do it for movie sets because idset is not a cXX field so is manually added to the query directly with the new id. cXX fields are loaded by a function that loops automatically and I can't change that part of the query. The alternative is to add another "rating_id" column and leave another cXX variable unused